README.md

```diff
@@ -42,9 +42,6 @@ brew install zstd
 
 # Decompress the dataset
 zstd -d esci-s/esci.json.zst
-
-# Filter for US locale and create en_esci.json
-grep '"locale":"us"' esci.json > en_esci.json
```

Comment:
@git-yogesh why did we remove this?

Reply:
English comprises more than 60% of the original data, so directly filtering would avoid saving and managing additional file for `en`.